recoverTranslations.js

```diff
@@ -0,0 +1,21 @@
+const fs = require("fs")
+
+let archivedMessages = fs.readdirSync("./z_archive/locales")
+let newSource = JSON.parse(fs.readFileSync("./locales/en.json", "utf-8"))
+
+archivedMessages.forEach((localeFile) => {
+  if (localeFile === "en.json") return // don't overwrite the source file
+
+  let oldMessages = JSON.parse(
+    fs.readFileSync(`./z_archive/locales/${localeFile}`, "utf-8")
+  )
+
+  let result = {}
+  for (let newSourceKey in newSource) {
+    if (oldMessages[newSourceKey]) {
+      result[newSourceKey] = oldMessages[newSourceKey]
+    }
+
+    fs.writeFileSync(`./locales/${localeFile}`, JSON.stringify(result))
+  }
+})
```

Comment:
Added this lil function to grab the handful of messages that are reused between v1 and v2. We can delete this later once we're pretty sure we've got all the old messages.

Later we can upload these existing translations to crowdin.